assets/app/view/game/part/track_node_path.rb

```diff
@@ -332,6 +334,17 @@ def load_from_tile
             @end_x,
             @end_y
           ) if @need_arc
+
+          return unless @path.parallel? && @begin_edge
```

Comment:
don't use unless with multiple

return if !parallel || !begin_edge

Reply:
Funny - I originally had this as a if block (w/o return) and rubocop wanted me to change it to be a "return unless", but "return if" works.